client/src/components/Login.js

```diff
@@ -3,39 +3,56 @@ import { Link, useNavigate } from "react-router-dom";
 import axios from 'axios';
 import { useUser } from "../contexts/UserProvider";
 import { BiArrowBack } from "react-icons/bi";
+import Alert from 'react-bootstrap/Alert';
+
 
 function Login() {
 	const [credentials, setCredentials] = useState({ email: "", password: "" });
-	const navigate = useNavigate();
+	const [show, setShow] = useState(false);
+	const navigate = useNavigate(); 
 	const { setUser } = useUser();
 
 	const handleSubmit = async (e) => {
 		e.preventDefault();
 
 		try {
-			const response = await axios.post(process.env.REACT_APP_SERVER_URL + "auth", credentials)
-			const { success, token, user } = response.data;
-
-			if (!success) return alert("Invalid credentials!");
+			const response = await axios.post(`${process.env.REACT_APP_SERVER_URL}auth`, {
+				"email": credentials.email,
+				"password" : credentials.password
+			})
+			const {token, user } = response.data;
 
 			localStorage.setItem('token', token);
 			localStorage.setItem('user', JSON.stringify(user));
 			setUser(user);
 			navigate("/");
 		} catch (err) {
 			console.log(err);
+			setShow(true)
 		}
 	}
 
 	const handleChange = (e) => {
 		setCredentials({ ...credentials, [e.target.name]: e.target.value })
 	}
 
+	function handleClose(){
+		setShow(false)
+	}
+
 	return (
 		<main className='login'>
 			<div className='header'>
 				<button className='btn back-btn' onClick={() => navigate(-1)}><BiArrowBack />Go Back</button>
 			</div>
+			{
+				show && (
+					<Alert variant='danger'>
+                        Invalid Credentials
```

Comment:
In the `auth.js` file we specify why the error occurs i.e. invalid credentials or incorrect password. Use that message here instead of hard coding it. Also, it would be easier to manage if you could change the line 47 of `auth.js` from 
```
return res.status(400).json({ error: [{error: 'Invalid password'}]})
```
to
```
return res.status(400).json({ error: [{msg: 'Invalid password'}]})
```

It would be easier to implement :)